components/emoji_picker/README.md

```diff
@@ -0,0 +1,168 @@
+We don't support all the emojis, like those which have multiple skin tone modifier.
```

Comment:
We still show these, but ONLY in the default skin tone right?

Reply:
great lets make that clear here so it doesn't sound like we are just removing valid emojis

Reply:
to be honest I dont know how to write it... because the sentence is correct... 
"like those which have multiple skin tone modifier." they are emojis with _toneX_toneY
the other are just default, they are not event default skin tone, just default.... 🤔